common/components/common/datetime/DateRangeSelectors.jsx

```diff
@@ -47,9 +48,9 @@ export class DateRangeSelectors extends React.PureComponent<Props, State> {
   }
  
   onChangeDateTime(start: string, end: string): void {
-    //If user entered start time after end time, reverse the two
+    //If user entered start time after end time, reverse the two (if option enabled)
     let _start:string, _end:string;
-    if(start < end || !start || !end) {
+    if(!this.props.enforceTimeOrder || !start || !end || start < end) {
```

Comment:
This is a small thing, but I can't envision a use case to keep the time order enforcement here. It might be worth it for code cleanliness to just remove the option? We're doing our checking in EventOverviewForm's checkFunc just like any other field, now.

Reply:
👍